blocks/identity-block/features/signup/default.jsx

```diff
@@ -114,7 +114,9 @@ const SignUp = ({ customFields, arcSite }) => {
 					},
 					{
 						email,
-					}
+					},
+					undefined,
+					{ rememberMe: true },
```

Comment:
I'm not sure if this has to be an object. @mattkim93 is this passed the same way as for the login method, or can it just be a boolean? Looking at the code it seems the latter may be true, but I'm not 100% certain.

Reply:
@vgalatro you might be right. I initially tried just using `undefined, rememberMe` but it didn't seem to be working for me locally. When I added it in as an object, it worked. I can retest it with just the boolean value again though

Reply:
@vgalatro updated! it seems to be working with just `true` instead of `{ rememberMe: true }`